src/interface/web/app/search/page.tsx

```diff
@@ -59,24 +98,131 @@ function getNoteTypeIcon(source: string) {
+                                                    file.file_name,
+                                                    file.raw_text,
+                                                )
+                                            }
+                                        >
+                                            <Download className="h-4 w-4" />
+                                        </Button>
+                                    </div>
+                                </DialogTitle>
+                            </DialogHeader>
+                            <ScrollArea className="h-[50vh]">
+                                <p className="whitespace-pre-wrap break-words text-sm font-normal">
+                                    {!selectedFileFullText && (
+                                        <InlineLoading
+                                            className="mt-4"
+                                            message={
+                                                "Loading"
+                                            }
+                                            iconClassName="h-5 w-5"
+                                        />
+                                    )}
+                                    {
+                                        selectedFileFullText
+                                    }
+                                </p>
+                            </ScrollArea>
+                        </DialogContent>
+                    </Dialog>
+                    <DropdownMenu>
+                        <DropdownMenuTrigger>
+                            <Button variant={"ghost"}>
+                                <DotsThreeVertical className="h-4 w-4" />
+                            </Button>
+                        </DropdownMenuTrigger>
+                        <DropdownMenuContent className="flex flex-col gap-0 w-fit">
+                            <DropdownMenuItem className="p-0">
+                                <Button
+                                    variant={"ghost"}
+                                    className="flex items-center gap-2 p-1 text-sm"
+                                    onClick={() => {
+                                        handleDelete(
+                                            file.file_name,
+                                        );
+                                    }}
+                                >
+                                    <Trash className="h-4 w-4" />
+                                    <span className="text-xs">
+                                        {isDeleting
+                                            ? "Deleting..."
+                                            : "Delete"}
+                                    </span>
+                                </Button>
+                            </DropdownMenuItem>
+                        </DropdownMenuContent>
+                    </DropdownMenu>
```

Comment:
Why have a dropdown for a single action, delete? Maybe the delete trash icon can be visible instead of the dropdown 3 dot menu on every card?

Reply:
Because the trash can icon takes up more space and adds visual clutter. Three dots is much more pleasant than a rubbish bin.

Also, I'd expect it to be used infrequently 🙂 .